csharp-basic-topics/ValidateNumberIsPositiveOrNegative/ValidateNumberIsPositiveOrNegative/Program.cs

```diff
@@ -0,0 +1,17 @@
+﻿using ValidateNumberIsPositiveOrNegative;
+
+public class Program
+{
+    private static void Main(string[] args)
+    {
+        int number = -42;
```

Comment:
Use var instead of int. Please check the coding guideline for more details.

Reply:
Updated